core/src/main/scala/kafka/server/AutoTopicCreationManager.scala

```diff
@@ -166,7 +179,32 @@ class DefaultAutoTopicCreationManager(
         debug(s"Auto topic creation completed for ${creatableTopics.keys}.")
```

Comment:
This message will probably cause confusion since `ClientResponse` may still indicate an error. How about we add the response to the log message?